src/LibClang-Core/CLTranslationUnit.class.st

```diff
@@ -104,7 +104,7 @@ CLTranslationUnit class >> unsavedFilesArray: anArrayOfAssociations [
 	| unsavedArray |
 
 	unsavedArray := FFIExternalArray 
-		newType: #CXUnsavedFileStruct 
+		externalNewType: #CXUnsavedFileStruct 
```

Comment:
are you sure you want this?
if you make it "external" you need to properly release it later or you will be leaking.